gbus/worker.go

```diff
@@ -207,6 +207,22 @@ func (worker *worker) isDead(delivery amqp.Delivery) bool {
 }
 
 func (worker *worker) invokeDeadletterHandler(delivery amqp.Delivery) {
```

Comment:
I think it would be better to reject the message in the recover function in the process message and not duplicate another recover function within the invokeDeadletterHandler.
In addition, it would probably be better to have the call to metrics.ReportRejected in the worker.reject function (and once there need to delete the call to metrics.ReportRejected from the processMessage function